infra/config-store/src/test/java/com/simprints/infra/config/store/models/FaceConfigurationTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.simprints.infra.config.store.models
+
+import com.google.common.truth.Truth
+import org.junit.Test
+
+class FaceConfigurationTest {
+    @Test
+    fun `should retrieve Rank One configuration  when RANK_ONE is requested `() {
+        val faceConfiguration = createConfiguration()
+        Truth
+            .assertThat(faceConfiguration.getSdkConfiguration(FaceConfiguration.BioSdk.RANK_ONE))
+            .isEqualTo(faceConfiguration.rankOne)
+    }
+
+    @Test
+    fun `should retrieve SimFace configuration  when NEC is requested `() {
```

Comment:
A minor confusion - is NEC here by accident?

```suggestion
    fun `should retrieve SimFace configuration  when SIM_FACE is requested `() {
```